### ISSUE TITLE
Learning Rate curve shape

### ISSUE THREAD

**Issue by cidelab** [Unity-Technologies/ml-agents#1755]
Good morning all,
The learning curve described in the _tensorboard_ graphs is very closer to a straight line.
I would like to change the decay function to a curve with a gentle slope. 
Which file should I check to change the decay function in the learning rate? (if is possible)

Thanks in advance
kind regards
Carlos.

**Comment by karta1297963** [comment]
https://github.com/Unity-Technologies/ml-agents/blob/d3f9fd63043f1c82790d3fe35ee07dc5ed1232b9/ml-agents/mlagents/trainers/ppo/models.py#L167

You can modify the 'power' parameter 
reference: https://www.tensorflow.org/api_docs/python/tf/train/polynomial_decay

**Comment by cidelab** [comment]
Thanks a lot @karta1297963 
The very best
C,
